src/civil/date.rs

```diff
@@ -1367,6 +1367,8 @@ impl Date {
     /// # Example: negative spans are supported
     ///
     /// ```
+    /// # // See: https://github.com/rust-lang/rust/pull/121364
+    /// # #![allow(unknown_lints, ambiguous_negative_literals)]
```

Comment:
All of those manual allow-attr can be replaced with a single [`#![doc(test(attr(...)))]`](https://doc.rust-lang.org/rustdoc/write-documentation/the-doc-attribute.html#testattr) attr: `#![doc(test(attr(allow(unknown_lints, ambiguous_negative_literals))))]`

Reply:
Oh wow, that's great to know! Thank you.